### PR TITLE
PublishTags

### LINKED ISSUE #505: It's not publishing on tags: why not?

### PR DESCRIPTION
**Pull request by ChristopherDavenport**
Should fix #505

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -52,6 +52,12 @@ lazy val commonSettings = Seq(
 
 ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.8")
 
+ThisBuild / githubWorkflowTargetTags ++= Seq("v*")
+
+// currently only publishing tags
+ThisBuild / githubWorkflowPublishTargetBranches :=
+  Seq(RefPredicate.StartsWith(Ref.Tag("v")))
+
```
Comment (rossabaker) [inline build.sbt:60]: These settings seem like they could go into the org plugin at some point?  Particularly in the Great Schism?

**Comment by rossabaker** [comment]
I'm not sure from the regenerated workflow if this is going to do anything?

**Comment by rossabaker** [comment]
Is [this](https://github.com/http4s/blaze/actions/runs/821819652/workflow#L64) the problematic line?

File: build.sbt (hunk 1 above)
Comment (zmccoy) [reply build.sbt:60]: Absolutely unrelated to what's being done here, but what is "The Great Schism"?

Comment (rossabaker) [reply build.sbt:60]: The break between the Roman Catholic and Eastern Orthodox Church in the 11th Century.

Alternatively, [this](https://github.com/http4s/http4s/discussions/4140).

Comment (zmccoy) [reply build.sbt:60]: Thanks!  For both even

**Comment by rossabaker** [comment]
I still don't understand what difference it makes from the yaml, but it matches the docs.  Maybe the plugin introspects the setting.

**Comment by rossabaker** [comment]
Drat.  Didn't work.

**Pull request merged**